.travis.yml

```diff
@@ -11,6 +11,9 @@ before_install:
 - npm i -g npm@8
 - npm --version
 
+install:
+- npm ci --legacy-peer-deps
```

Comment:
There are some dependency stuff that google seems to say to fix by using this flag, but not sure if there's any real problems with doing so.